src/test/kotlin/no/nav/helsearbeidsgiver/utils/InnsendingUtilsTest.kt

```diff
@@ -0,0 +1,50 @@
+package no.nav.helsearbeidsgiver.utils
+
+import io.kotest.matchers.shouldBe
+import no.nav.helsearbeidsgiver.forespoersel.Status
+import no.nav.helsearbeidsgiver.utils.test.wrapper.genererGyldig
+import no.nav.helsearbeidsgiver.utils.wrapper.Fnr
+import no.nav.helsearbeidsgiver.utils.wrapper.Orgnr
+import org.junit.jupiter.api.Test
+import java.util.UUID
+
+class InnsendingUtilsTest {
```

Comment:
Finnes det positive tester for dette et sted?

Reply:
Nei, bare den første: validerOKMotForespoersel() 
Det var ingen (eksplisitte) tester for denne klassen i det hele tatt la jeg merke til - men noe av logikken, spesielt happy-case var fsv dekket i integrasjonstester. Men foretrekker små unit-tester der det er mulig.

Reply:
Jeg så på halve filen før standup og halve etter, så jeg glemte at `validerOKMotForespoersel` fantes 🥇🐟 Den dekker jo hele happy casen 👍